### ISSUE TITLE
Invalid supported color modes

### ISSUE THREAD

**Issue by 8none1** [raulgbcr/lednetwf_ble#22]
The latest version (not yet released) triggers this error:

```
2024-06-21 15:22:03.265 WARNING (MainThread) [homeassistant.components.light] None (<class 'custom_components.lednetwf_ble.light.LEDNETWFLight'>) sets invalid supported color modes {<ColorMode.BRIGHTNESS: 'brightness'>, <ColorMode.COLOR_TEMP: 'color_temp'>, <ColorMode.HS: 'hs'>}, this will stop working in Home Assistant Core 2025.3, please create a bug report at https://github.com/raulgbcr/lednetwf_ble/issues
```

This means we need to port everything to using HS colorspaces instead of RGB.

I expect this is because it will remove the need to track colour and brightness separately.  It makes sense but it will be quite a significant change.

**Comment by raulgbcr** [comment]
Yeah, I had that in the radar, there are some very angry looking comments on the places this has to be changed that I identified. It's on the list of things to do some time in the future, unless you wanna go ahead and tackle it hehe

**Comment by raulgbcr** [comment]
Added a fix for this on 00c54de698d3f75ccfedd2af4f536ab4edc6ea26 on a new test branch, initial testing seems promising on my side, hopefully was a minor thing I introduced on the 2024.2 update and should be fixed with this

**Comment by 8none1** [comment]
Sorry, I haven't forgotten about this, just been a bit busy with other things.  I will pick this up soon.
